src/test/java/com/automationPractice/pages/ContactUsPage.java

```diff
@@ -11,13 +11,47 @@ public ContactUsPage (WebDriver webDriver){
 
     By Form_principal = By.id("center_column");
     By Tittle_ContactUsForm = By.xpath("//div[@id='center_column']//h1");
+    By Select_subject = By.xpath("//select[@id='id_contact']");
```

Comment:
Why do you use By.xpath instead of By.id for the locator.

Reply:
Because the id "id_contact" isn't unique. Exist three elements in the page with the same id name.

Reply:
could you try using name? 
![image](https://user-images.githubusercontent.com/21041109/66884123-ccffea00-ef95-11e9-9ad4-606ad36cd691.png)

Reply:
Ok, I changed the search criteria: "xpath" by "name".